cli/internal/helm/client.go

```diff
@@ -168,47 +176,15 @@ func (c *Client) GetValues(release string) (map[string]any, error) {
 	return values, nil
 }
 
-// ApplyCRD updates the given CRD by parsing it, querying it's version from the cluster and finally updating it.
-func (c *Client) ApplyCRD(ctx context.Context, rawCRD []byte) error {
-	crd, err := parseCRD(rawCRD)
-	if err != nil {
-		return fmt.Errorf("parsing crds: %w", err)
-	}
-
-	clusterCRD, err := c.crdClient.ApiextensionsV1().CustomResourceDefinitions().Get(ctx, crd.Name, metav1.GetOptions{})
-	if err != nil {
-		return fmt.Errorf("getting crd: %w", err)
-	}
-	crd.ResourceVersion = clusterCRD.ResourceVersion
-	_, err = c.crdClient.ApiextensionsV1().CustomResourceDefinitions().Update(ctx, crd, metav1.UpdateOptions{})
-	return err
-}
-
-// parseCRD takes a byte slice of data and tries to create a CustomResourceDefinition object from it.
-func parseCRD(crdString []byte) (*v1.CustomResourceDefinition, error) {
-	sch := runtime.NewScheme()
-	_ = scheme.AddToScheme(sch)
-	_ = v1.AddToScheme(sch)
-	obj, groupVersionKind, err := serializer.NewCodecFactory(sch).UniversalDeserializer().Decode(crdString, nil, nil)
-	if err != nil {
-		return nil, fmt.Errorf("decoding crd: %w", err)
-	}
-	if groupVersionKind.Kind == "CustomResourceDefinition" {
-		return obj.(*v1.CustomResourceDefinition), nil
-	}
-
-	return nil, errors.New("parsed []byte, but did not find a CRD")
-}
-
-// updateCRDs walks through the dependencies of the given chart and applies
+// updateOperatorCRDs walks through the dependencies of the given chart and applies
 // the files in the dependencie's 'crds' folder.
 // This function is NOT recursive!
 func (c *Client) updateCRDs(ctx context.Context, chart *chart.Chart) error {
 	for _, dep := range chart.Dependencies() {
 		for _, crdFile := range dep.Files {
 			if strings.HasPrefix(crdFile.Name, "crds/") {
-				c.log.Debugf("Updating crd: %s", crdFile.Name)
-				err := c.ApplyCRD(ctx, crdFile.Data)
+				c.log.Debugf("updating crd: %s", crdFile.Name)
```

Comment:
```suggestion
				c.log.Debugf("Updating crd: %s", crdFile.Name)
```
I'm not sure if we have defined logging guidelines somewhere, but generally log messages start Uppercase and end without a `.`